semantic-conventions/hs-opentelemetry-semantic-conventions.cabal

```diff
@@ -0,0 +1,73 @@
+cabal-version: 2.4
+
+name: hs-opentelemetry-semantic-conventions
+version: 1.24.0.0
+author: Kazuki Okamoto (岡本和樹)
+maintainer: kazuki.okamoto@herp.co.jp
+synopsis: OpenTelemetry Semantic Conventions for Haskell
+description:
+  OpenTelemetry Semantic Conventions for Haskell is a library that is automatically generated
+  based on [semantic-conventions](https://github.com/open-telemetry/semantic-conventions/) v1.24.
+
+  This module's version A.B.X.Y means that A.B is the version of the original semantic-conventions,
```

Comment:
```suggestion
  This package's version A.B.X.Y means that A.B is the version of the original semantic-conventions,
```

Reply:
Note that this strategy doesn't let us do major releases of this package _without_ a corresponding version bump of the semantic conventions, right? That seems awkward. 

In the past I've mostly just given up on this approach and given the Haskell package its own versioning scheme. That's what we do with the OTLP package in this repository, for example.

Reply:
Hmm how about $(A \times 100 + B).x.y.z$?

I think users want to find the newer version of hs-semconv that compatible with a semconv version. When hs-semconv has its own versioning, A conversion table is necessary to find compatible versions and also version bounds, like `= 1029.1.*`, cannot be used.

Reply:
I guess I am unsure whether people actually really need what you suggest. My experience (e.g. with the LSP protocol package) has been that specification versions supported by a library change quite rarely, and people either tend to want "the latest" (in which case they don't care), or unusually "a very specific version", in which case they have to pin a version of the package anyway.

As for a conversion table, I think it would be good enough to just list the supported spec version in the package description? Then it doesn't take too long to click through the versions on Hackage to find the one that you want?

Reply:
OK. Just follows PVP.